db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java

```diff
@@ -52,6 +52,9 @@
 	protected static final int		SYSCOLUMNS_CORE_NUM = 2;
 	protected	static final int		SYSSCHEMAS_CORE_NUM = 3;
 	protected static final int        NUM_CORE = 4;
+	// NOTE: JC - When adding a new sys table, this
+	// number will need to be increased in BOTH places.
+	public static final long FIRST_USER_TABLE_NUMBER = 1458;
```

Comment:
This constant is also defined in DataDictionary, I think we should use just that one and remove this.